apps/localizer/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy ca-certificates curl gpg lsb-release \
+    unzip libgomp1 libglib2.0-0 libdbus-1-3 libfftw3-single3 \
+    libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 \
+    libxcb-render-util0 libxcb-xinerama0 libxcb-xkb1 libxkbcommon-x11-0 \
+    libxcb-xinput0 libxcb-shape0 fontconfig && \
+    curl -sSOL https://github.com/CRNL-Eduwell/Localizer/releases/download/V${APP_VERSION}/Localizer.${APP_VERSION}.linux64.zip && \
+    mkdir ./install && \
+    unzip -q -d ./install Localizer.${APP_VERSION}.linux64.zip && \
+    chmod -R 757 ./install/Localizer.${APP_VERSION}.linux64 && \
```

Comment:
757 is a strange one.

Reply:
So is the dev to be honest. 
Joke aside I probably uploaded the build and it's config file with weird rights and this was my lazy solution to not redo a build